graphite/error.go

```diff
@@ -0,0 +1,46 @@
+package graphite // import "bosun.org/graphite"
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+func NewParseError(message string, err error, url url.URL) error {
```

Comment:
I do not think either of these methods are needed and will have to be really convinced before I ok them. This one especially is so simple that it does not warrant existing.